src/mixins/render-vega-lite/Definitions/Encoding/Scale/Discretizing/DiscretizingScale.js

```diff
@@ -109,6 +109,6 @@ export default class DiscretizingScale extends ScaleDefinitionObject {
    */
   // eslint-disable-next-line no-unused-vars
   _materializeExtraVegaScaleProps(prop_descriptor, vega_scale_object) {
-    assert(false)
```

Comment:
this was unnecessary and only cause headaches to those of you trying to use quantize scales with auto domains.